tests/Zend/TranslateTest.php

```diff
@@ -716,7 +716,8 @@ public function testIgnoreMultipleDirectories()
             'auto',
             array(
                 'scan' => Zend_Translate::LOCALE_FILENAME,
-                'ignore' => array('.', 'ignoreme', 'LC_TEST')
+                'ignore' => array('.', 'ignoreme', 'LC_TEST'),
+                'disableNotices' => true,
```

Comment:
This just suppresses an E_USER_NOTICE that this path triggers, doesn't seem that test was affected either way, it's just that Notices now cause tests to fail.